acceptance/bundle/trampoline/warning_message_with_new_spark/test.toml

```diff
@@ -1,6 +1,6 @@
 # Since we use existing cluster id value which is not available in cloud envs, we need to stub the request
 # and run this test only locally
-LocalOnly = true
+Cloud = false
```

Comment:
All these layers of override can be difficult to track. Could you add some functionality to see the resolved configuration for a test case or directory as a follow-up?

Reply:
I don't think there is an easy way, so you'll have to track those manually. For a given boolean option you only need to know the closest setting though, not the whole chain.